### DISCUSSION TITLE
Django + Angular integration.

### DISCUSSION THREAD

**Discussion by Rahma-AlWadhahi** [EddieHubCommunity/support#709] [Tips and Tricks]
A question has been raised by @jveer634 in discord about how to deploy a project that has both Angular and Django together in the same project. Based on @Vyvy-vi suggestion to open a discussion about integrating Django and Angular together.

I will share a couple articles about it below:
- [Angular and Django Integration in one project](https://hackernoon.com/angular-and-django-integration-in-one-project-ip7k3yuh) 
- [How to Deploy a Static Site from GitHub with DigitalOcean Application Platform Quickstart](https://www.digitalocean.com/community/tutorials/how-to-deploy-a-static-site-from-github-with-digitalocean-app-platform-quickstart)

**Comment by eddiejaoude** [comment]
Thank you for sharing @Rahma-AlWadhahi 👍 

Note: I changed the category to `tips and tricks`

**Reply by Rahma-AlWadhahi** [reply]
Oh thank you @eddiejaoude. Are we sharing links on how to do stuff in `tips and tricks` category?
This will help us to specify the right category from the beginning 😄

**Reply by eddiejaoude** [reply]
Yep, GitHub links can be shared in Discord. Having them here helps them not to get lost in the Discord history, then we can reshare the links from GitHub again when people ask about a similar topic
